commons/passive/garmin/garmin_generic_heartrate.avsc

```diff
@@ -0,0 +1,46 @@
+{
+   "name":"GarminGenericHeartRate",
+   "type":"record",
+   "namespace":"org.radarcns.passive.garmin",
+   "doc":"Provides the data about real time Heart rate data.",
```

Comment:
How are high, resting and low heart rate defined? Is "daily" the average over the last 24 hours, since midnight, or is there some proprietary calculation done by Garmin? Please put in the docs.

Reply:
Reply from Garmin team - High and low heart rate are simply the highest and lowest individual HR values recorded daily up to a given point, when the RestingHeartRate message is written.  Resting heart rate itself is calculated algorithmically based on recordings taken during periods of sustained low motion.

Reply:
“Daily" values provided by the SDK are always reset at midnight of the local user-day.  This is true for Heart Rate metrics and cumulative steps counts.

Reply:
Updated these definitions in document. will be available in latest commit.